sync/bi_sync_mysql.go

```diff
@@ -0,0 +1,282 @@
+package sync
+
+import (
+	"bytes"
+	"fmt"
+	"github.com/2tvenom/myreplication"
+	_ "github.com/go-sql-driver/mysql"
+	"github.com/go-xorm/xorm"
+	"github.com/spf13/viper"
+	"log"
+	"xorm.io/core"
+)
+
+type DBConfig struct {
+	Host     string `mapstructure:"host"`
+	Port     int    `mapstructure:"port"`
+	Username string `mapstructure:"username"`
+	Password string `mapstructure:"password"`
+	Database string `mapstructure:"database"`
+	Base     string `mapstructure:"base"`
+}
+
+type Config struct {
+	Databases []DBConfig `mapstructure:"mysql"`
+}
+
+var (
+	dbTables   = make(map[string]*core.Table)
+	db1        DBConfig
+	db2        DBConfig
+	ConfigData *Config
+)
+
+func initConfig() {
+
+	// set config file
+	viper.SetConfigType("yaml")
```

Comment:
No need to use yaml, define conf.go:

```go
var dataSourceName1 = "root:123456@tcp(localhost:3306)/"
var dataSourceName2 = "root:123456@tcp(localhost:3307)/"
```

Like: https://github.com/casdoor/casdoor/blob/e71e41b3433e1dae861cbf6530f3aefcb1393d47/conf/app.conf#L6